test/run.d

```diff
@@ -145,7 +181,29 @@ auto predefinedTargets(string[] targets)
         return testPath(dir).dirEntries("*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
     }
 
-    Appender!(string[]) newTargets;
+    static Target createUnitTestTarget()
+    {
+        Target target = {
+            args: [getEnvironment["DMD"], "-run", "unit_test_runner.d"]
+        };
+
+        return target;
+    }
```

Comment:
This won't play nice with rebuilds.

Reply:
Any suggestions?

Reply:
It's not that important as we currently don't have many tests, but in the future we probably have to do:
- write a file after every invocation
- lastEdit  = MAX(timestamp of src/*.d)
- compare lastEdit with the timestamp of the written file

(that's how the current testsuite does it, except that it checks the timestamp of the DMD binary which wouldn't be helpful here.)

Reply:
That could work, but it would complicate the unit test runner even more. It would also need to check `unit/*.d`.